app/models/events/answer_set.rb

```diff
@@ -1,24 +1,26 @@
 module Events
   class AnswerSet < Event
-    event_attributes question_id: :string,
+    event_attributes domain_id: :string,
```

Comment:
Dit is omdat je anders later niet weet bij welk domein een vraag hoort. Dat is nuttige informatie bij het tonen van de antwoorden in het eindrapport.